src/components/seekbar/seekbar.js

```diff
@@ -25,6 +26,7 @@ const COMPONENT_NAME = 'SeekBar';
   mapStateToProps,
   bindActions(actions)
 )
+@withPlayer
```

Comment:
although it doesn't inherit from baseComponent you might consider with logger here